test/karma.conf.js

```diff
@@ -95,7 +95,7 @@ module.exports = function setKarmaConfig(config) {
           os: 'OS X',
           os_version: 'Sierra',
           browser: 'Chrome',
-          browser_version: '49.0',
+          browser_version: '45.0',
```

Comment:
Isn't the bot running `chrome@41`?

Reply:
Sinon.js doesn't support it. It's also why we don't run the tests in IE11.

Reply:
As I interpret their readme it should support IE11. But chrome only at 69 which doesn't make much sense to me. I'm probably misunderstanding their saucelabs badge.

So we are basically not doing any tests at all in IE11?

Reply:
e2e Chrome support starts at v49. JavaScript support starts at v41. e2e IE support starts at v11.

Reply:
I wish we could make the tests pass in IE11.